cogs5e/lookup.py

```diff
@@ -193,7 +193,7 @@ async def racefeat(self, ctx, *, name: str):
         )
         return await self._racefeat(ctx, result)
 
-    @slash_lookup.sub_command(name="racefeat", description="Looks up a species feature.", aliases=["speciesfeat"])
+    @slash_lookup.sub_command(name="racefeat", description="Looks up a species feature.")
```

Comment:
I might have considered just renaming these commands as opposed to duplicating them-